src/styles.scss

```diff
@@ -45,7 +45,19 @@ html,
 body {
   height: 100%;
 }
+
 body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+
+.footer {
+  text-align: center;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 40px;
+  background-color: #5460ac;
+  color: rgb(9, 2, 24);
+  margin: 0px;
+}
```

Comment:
Currently, the footer is looking kinda off
![image](https://user-images.githubusercontent.com/69753899/195454245-0baadd6d-3e0e-46de-8e43-8befe09ba124.png)

try adding

`display: flex`, `justify-content: center`, `align-items: center` to `.footer` 
and remove the bottom margin from '.footer p' ;)

Reply:
I am wondering, after this changes, do I need text-align: center when I use align-items: center?

Reply:
@Iwona007 you don't. `justify-content: center` will basically already align text in center